docs/website/docs/general-usage/customising-pipelines/removing_columns.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Removing columns
+description: Removing columns by passing list of column names
+keywords: [deleting, removing, columns, drop]
+---
+
+# Removing columns
+
+Removing columns before loading data into a database is a reliable method to eliminate sensitive or
+unnecessary fields. For example, in the given scenario, a source is created with a "country_id" column,
+which is then excluded from the database before loading.
+
+Let's create a sample pipeline demonstrating the process of removing a column.
+
+1. Create a source function that creates dummy data as follows:
+
+   ```python
+   import dlt
+
+   @dlt.source
+   def dummy_source(prefix: str = None):
+       #This function creates a dummy data source.
+       @dlt.resource(write_disposition='replace')
+       def dummy_data():
+             for _ in range(3):
```

Comment:
usually people use _ if they iterate but do not use the value of this variable.
like:
```
for _ in range(3): 
    print("I love dlt")
```
but you use it. so it will be less confusing if you name this variable not _ but i
```
for i in range(3): 
   print(i)

```

Reply:
done